src/test/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServerTest.java

```diff
@@ -53,6 +52,15 @@
   public void setUp() {
     devServer = new CloudSdkAppEngineDevServer(sdk);
   }
+  
+  @Test
```

Comment:
I believe you have a reason you don't prefer `@Test(expected = NullPointerException.class)` here, so I'm good.